### ISSUE TITLE
new version number breaks dependencies

### ISSUE THREAD

**Issue by LlubNek** [SilentChaos512/Silent-Gear#347]
## Versions

managear-1.16.3-2.2.3.jar
- **Silent Gear**: SilentGear-1.16.3-2.6.8.jar
- **Silent's Gems**: SilentGems-1.16.3-3.7.11+114.jar
- **Silent Lib**: silent-lib-1.16.3-4.9.4+68.jar
- **Forge**: 36.0.46
- **Modpack**: N/A 
- **Optifine Installed**: Yes 

## Expected Behavior

normal startup

## Actual Behavior

dependency error from managear since forge can't parse your version number correctly
```
-- MOD managear --
Details:
	Mod File: managear-1.16.3-2.2.3.jar
	Failure message: Mod managear requires silentgear 2.3.3 or above
		Currently, silentgear is 1.16.3-2.6.8
	Mod Version: 1.16.3-2.2.3
	Mod Issue URL: https://github.com/Partonetrain/managear/issues
	Exception message: MISSING EXCEPTION MESSAGE
```

## Links/Images

![image](https://user-images.githubusercontent.com/313762/110691664-6c3d0700-81b3-11eb-9757-72e5b541ec6c.png)
[crash-2021-03-10_15.03.03-fml.txt](https://github.com/SilentChaos512/Silent-Gear/files/6118620/crash-2021-03-10_15.03.03-fml.txt)
[latest.log](https://github.com/SilentChaos512/Silent-Gear/files/6118623/latest.log)

## Steps to Reproduce the Problem

1. install silent mods
2. install managear
3. start minecraft

**Comment by SilentChaos512** [comment]
Issue corrected in version 2.6.8.1 (https://www.curseforge.com/minecraft/mc-mods/silent-gear/files/3234856)

**Comment by LlubNek** [comment]
looks good
